python-checks/src/test/resources/checks/uselessStatement/uselessStatementImport.py

```diff
@@ -0,0 +1,4 @@
+from uselessStatementImported import ClassWithProperty
+
+ClassWithProperty().method # Noncompliant
+ClassWithProperty().my_property
```

Comment:
It would be nice to see a comment here explaining that this case is ok, and exactly what we test here